RefrigeratorExe/RefrigeratorExe/Refrigerator.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Refrigerator :IComparable<Refrigerator>
+    {
+        private string _model;
+
+
+        public Guid id { get; }
+        public string Model
+        {
+            get
```

Comment:
no need to hold _model and Model. This is redundant property and increases the program memory.
Hold this property only:

public string Model { get; }

Then in the constructor it is a good practice to throw exceptions before setting its value.